app/views/layouts/_edit_form_buttons.html.haml

```diff
@@ -8,8 +8,8 @@
 - serialize            ||= false
 - continue_button      ||= nil
 - restful              ||= false
-- show_cancel_button = %w(user_edit user_update menu_tree zone_edit zone_update ldap_seq_edit rbac_group_edit rbac_group_update rbac_group_field_changed category_edit category_update label_tag_mapping_edit label_tag_mapping_update timeprofile_edit timeprofile_update timeprofile_copy)
-- force_cancel_button ||= false
+- no_cancel_button     ||= false
```

Comment:
This might possibly break things, what about all the other pages which this list refers to?

Reply:
All the pages in the list no longer use this form buttons partial, they either use x_form_buttons or form_buttons, most of those are actions from OPS explorer.